var/spack/repos/builtin/packages/py-dcm2bids/package.py

```diff
@@ -13,11 +13,15 @@ class PyDcm2bids(PythonPackage):
     homepage = "https://github.com/unfmontreal/Dcm2Bids"
     pypi = "dcm2bids/dcm2bids-2.1.9.tar.gz"
 
+    version("3.1.0", sha256="53a8a177d556df897e19d72bd517fdae0245927a8938bb9fbbd51f9f33f54f84")
     version("2.1.9", sha256="d962bd0a7f1ed200ecb699e8ddb29ff58f09ab2f850a7f37511b79c62189f715")
 
     depends_on("python@3.7:", type=("build", "run"))
```

Comment:
Shouldn't this be 3.8 now? >3.7

Reply:
Thanks. Didn't see that because they just exchanged `>=` with `>`.